app/Domain/Goalcanvas/Controllers/Dashboard.php

```diff
@@ -305,6 +305,18 @@ public function run()
                 }
             }
 
+            // request
+            // $filter['status'] = request('filter_status', session('filter_status', 'all'));
+            // session(['filter_status' => $filter['status']]);
+            // $filter['relates'] = request('filter_relates', session('filter_relates', 'all'));
+            // session(['filter_relates' => $filter['relates']]);
+
+            $filter['status'] = $_GET['filter_status'] ?? (session("filter_status") ?? 'all');
```

Comment:
instead of `$_GET`/`$_POST` please use `$params`

Reply:
will done in the controller service refactor branch